WikiTemplate/Crypto/wikitemplate-cryptoDesktop.md

```diff
@@ -0,0 +1,151 @@
+
+## Onboarding
+
+- [ ] Verify loading `brave://wallet` on a clean profile shows onboarding flow
+- [ ] Verify you are able to complete the onboarding flow to Ethereum create wallet by default
+- [ ] Verify you are able to complete the backup seed phrase during onboarding itself
+- [ ] Verify you can skip the onboarding from backup wallet screen and still create a wallet
+- [ ] Verify when a Dapp interacts without creating wallet opens a new tab and shows onboarding
+
+## Importing Wallet
+
+- [ ] Verify with MetaMask extension installed and wallet created, visiting `brave://wallet/crypto/onboarding/import-or-restore` shows option to import account from MetaMask during onboarding
+- [ ] Verify error message is shown when trying to use same password as MetaMask and it doesn't meet Brave Wallet's password criteria
+- [ ] Verify able to successfully import MetaMask wallet into Brave Wallet
+- [ ] Verify with Crypto Wallet Extension installed and wallet created, visiting `brave://wallet/crypto/onboarding/import-or-restore` shows option to import account from old Crypto Wallet Extension
+- [ ] Verify error message is shown when trying to use same password as Crypto Wallet Extension and it doesn't meet Brave Wallet's password criteria
+- [ ] Verify able to successfully import Crypto Wallet Extension wallet into Brave Wallet
+
+## Panel
+
+- [ ] Verify when wallet is not created opening the wallet panel shows `Learn More` message Clicking on it opens `brave://wallet` in a new tab
+- [ ] Verify when wallet is created, opening the panel shows default account and selected network and account balance
+- [ ] Verify Buy/Send/Swap buttons are available and loads corresponding screens to perform action
+- [ ] Verify transaction history button is shown and lists recent transactions for selected network when clicked
+- [ ] Verify changing network in panel updates the corresponding account in panel and in portfolio page
+- [ ] Verify changing account in panel updates the corresponding network in panel and in portfolio page
+- [ ] Verify when a Dapp is connected to wallet account, panel shows a `Connected` state notification
+- [ ] Verify when a Dapp connection is revoked, panel shows a `Disconnected` state notification
+- [ ] Verify selecting `Connected Sites` from panel menu loads `brave://settings/content/<Network>` for Ethereum/Solana based on selected account
+- [ ] Verify selecting `View on Block explorer` from panel loads the corresponding account's details on the network block explorer
+- [ ] Verify selecting `Help Center` opens a new tab and loads `https://support.brave.com/hc/en-us/articles/4415497656461-Brave-Walllet-FAQ`
+- [ ] Verify you are able to lock the wallet from panel menu
+- [ ] Verify clicking on the expand button on the top-left of panel loads the wallet page in a new tab
```

Comment:
s/expand button/`expand` (or maybe it's `Expand`?) button?

Reply:
This is just an icon there is no text so no need to capitalize